.golangci.yml

```diff
@@ -60,15 +60,28 @@ linters-settings:
       - opinionated
     disabled-checks:
       - unnamedResult
+      - whyNoLint # TODO enable.
   funlen:
     lines: 60
     statements: 40
+  wsl:
+    strict-append: true
+    allow-assign-and-call: true
+    allow-multiline-assign: true
+    allow-case-traling-whitespace: true
+    allow-cuddle-declarations: false
 
 linters:
   enable-all: true
   disable:
     - maligned
     - prealloc
+    - testpackage
```

Comment:
seems like we have some TODOs to enable more linters :).

Reply:
yes